.github/workflows/remove-labels.yml

```
@@ -0,0 +1,50 @@
+name: Remove outdated labels
+on:
+  pull_request_target:
+    types:
+      - closed
+  issues:
+    types:
+      - closed
+
+jobs:
+  remove-merged-pr-labels:
+    name: Remove merged pull request labels
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mondeja/remove-labels-gh-action@v1.0.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          labels: |
+            awaiting reply
+            changes requested
+            duplicate
+            in discussion
+            invalid
+            out of scope
+            pending
+            won't add
+
+  remove-closed-pr-labels:
+    name: Remove closed pull request labels
+    if: github.event_name == 'pull_request_target' && (! github.event.pull_request.merged)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mondeja/remove-labels-gh-action@v1.0.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          labels: in discussion
```

Thinking about it more, should we include `pending` here as well?

```suggestion
          labels: |
            in discussion
            pending
```

---

Absolutly!